File: 2819-remove-trailing-zeros-from-a-string/2819-remove-trailing-zeros-from-a-string.java
class Solution {
    public String removeTrailingZeros(String num) {
        String res = "";
        for(int i = num.length() - 1; i >= 0; i--)
        {
            if(num.charAt(i) != '0')
            {
                res += num.substring(0,i+1);
                break;
            }
        }

        return res;
    }
}

File: 2819-remove-trailing-zeros-from-a-string/2819-remove-trailing-zeros-from-a-string.cpp
class Solution {
public:
    string removeTrailingZeros(string num)
    {
        int i = 0;
     for(i = num.length() - 1; i >= 0; i--)
     {
         if(num[i] != '0')
         {
             break;
         }
     } 

     return num.substr(0,i+1);  
    }
};